cpp/locality/NeighborList.h

```diff
@@ -112,7 +112,21 @@ class NeighborList
     template<typename Iterator> unsigned int filter(Iterator begin);
     //! Remove bonds in this object based on minimum and maximum distance
     //  constraints. Returns the number of bonds removed.
-    unsigned int filter_r(float r_max, float r_min = 0);
+    unsigned int filter_r(float r_max, float r_min = 0){
```

Comment:
This is the header file. The implementation of this method is in `NeighborList.cc`, and the validation logic has been added there already. Please revert this change, since only the method declaration must occur in the header.